src/nvme/types.h

```diff
@@ -5192,6 +5192,20 @@ struct nvme_lba_status {
 	struct nvme_lba_status_desc descs[];
 };
 
+/**
+ * enum nvme_lba_status_cmpc - Get LBA Status Command Completion Condition
+ * @NVME_LBA_STATUS_CMPC_INCOMPLETE:	Command completed, but additional LBA Status
+ *					Descriptor Entries are available to transfer
+ *					or scan did not complete (if ATYPE = 10h)
+ * @NVME_LBA_STATUS_CMPC_COMPLETE:	Completed the specified action over the number
+ *					of LBAs specified in the Range Length field and
+ *					transferred all available LBA Status Descriptors
+ */
+enum nvme_lba_status_cmpc {
```

Comment:
Should we also add the `no indication of completion condition` value?

Reply:
Yes, it may require. Will add it in the enum.
Thanks!